package-lock.json

```diff
@@ -0,0 +1,3415 @@
+{
+  "name": "j40_map",
+  "lockfileVersion": 2,
```

Comment:
`lockfileVersion": 2`

Reply:
This was from merging with another fix; my latest version just includes the score notebook